packages/commonwealth/client/scripts/state/api/trpc/subscription/useCreateCommunityAlertMutation.ts

```diff
@@ -1,5 +1,12 @@
+import { queryClient } from 'state/api/config';
 import { trpc } from 'utils/trpcClient';
 
 export function useCreateCommunityAlertMutation() {
-  return trpc.subscription.createCommunityAlert.useMutation();
+  return trpc.subscription.createCommunityAlert.useMutation({
+    onSuccess: async () => {
+      await queryClient.invalidateQueries({
+        queryKey: trpc.subscription.getCommunityAlerts.getQueryKey({}),
+      });
```

Comment:
lets use trpc.useutils for invalidating trpc queries

you can find example [here](https://github.com/hicommonwealth/commonwealth/blob/6ff154894b84fedaabff9bbb78a247fda147cc93/packages/commonwealth/client/scripts/state/api/contests/cancelContest.ts#L4)

```
import { trpc } from 'utils/trpcClient';

const useCancelContestMutation = () => {
  const utils = trpc.useUtils();

  return trpc.contest.cancelContestMetadata.useMutation({
    onSuccess: async () => {
      await utils.contest.getAllContests.invalidate();
    },
  });
};

export default useCancelContestMutation;
```